NAMESPACE

```diff
@@ -25,6 +25,10 @@ export(dropdownDivider)
 export(dropdownItem)
 export(dropdownItemList)
 export(flipBox)
+export(flip_box)
```

Comment:
Here it is snakeCase. User may get confused

Reply:
right, the reasoning behind me keeping snake_case was so I did not overwrite your original flipBox code.

However, I went ahead and made your requested changes (better late than never!)

See [my latest commit](https://github.com/RinteRface/shinydashboardPlus/pull/77/commits/62a1af6ac560769082fa4b4d220de22994a84715).

Additionally, I added an input alias so the user can call input$< id >_value to access the card's state where 0=front, 1=back, and NULL = initial state. 

However, the card itself does not utilize a shiny input binding I simply used `Shiny.setInputValue` on a new id that adds a "_value" suffix to the assigned flipbox id.

This is displayed in the demo app nicely via `flipBoxDemo()`.

![flipBoxDemo](https://user-images.githubusercontent.com/32652297/95931795-185efb00-0d98-11eb-821a-85e73a556c5e.gif)